uber/templates/preregistration/duplicate.html

```diff
@@ -3,7 +3,8 @@
 {% block backlink %}{% endblock %}
 {% block content %}
 
-    <div class="panel panel-default">
+<div class="masthead"></div>
+<div class="panel panel-default">
```

Comment:
Ok, just went back and looked, and it's correct as-is (the div is closed at the bottom of the template).